src/Mod/Fem/femsolver/settings.py

```diff
@@ -20,11 +20,12 @@
 # *                                                                         *
 # ***************************************************************************
 
-
-__title__ = "settings"
+__title__ = "FreeCAD FEM settings"
```

Comment:
is `settings` correct? or can there be more elaboration ?

Reply:
FreeCAD FEM solver settings